### DISCUSSION TITLE
What is your favorite app?

### DISCUSSION THREAD

**Discussion by gotutiyan** [NAIST-SE/AttractingContributors#51] [General]
https://github.com/gotutiyan/AttractingContributors_smartphone_app

**Comment by satokibi44** [comment]
added!
Cloud you contribute my repo?
https://github.com/satokibi44/AttractingContributors-Anime

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Comment by vittunyuta-aom** [comment]
added please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by gotutiyan** [comment]
age

**Comment by 278Mt** [comment]
added! and plz help my favorite movies!
https://github.com/278Mt/AttractingContributors
thx;-)

**Comment by sergiodl-naist** [comment]
This one is very convenient! To transform pictures of book pages into PDF's, with very good defaults for the preprocessing.
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
